src/LayoutManager/LayoutStrategies/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { Point } from '../../Point';
+import type { FabricObject } from '../../shapes/Object/FabricObject';
+import { cos } from '../../util/misc/cos';
+import { degreesToRadians } from '../../util/misc/radiansDegreesConversion';
+import { sin } from '../../util/misc/sin';
+import type { StrictLayoutContext } from '../types';
+
+export const getObjectSizeVector = (object: FabricObject) => {
+  const sizeVector = object._getTransformedDimensions().scalarDivide(2);
+  if (object.angle) {
+    const rad = degreesToRadians(object.angle),
+      sine = Math.abs(sin(rad)),
+      cosine = Math.abs(cos(rad)),
+      rx = sizeVector.x * cosine + sizeVector.y * sine,
+      ry = sizeVector.x * sine + sizeVector.y * cosine;
+    return new Point(rx, ry);
+  }
+  return sizeVector;
+};
+
+export const getObjectBounds = (object: FabricObject) => {
+  const objCenter = object.getRelativeCenterPoint();
+  const sizeVector = getObjectSizeVector(object);
+  return [objCenter.subtract(sizeVector), objCenter.add(sizeVector)];
+};
+
+export const shouldPerformLayout = (context: StrictLayoutContext) => {
+  return (
+    context.type === 'initialization' ||
+    context.type === 'imperative' ||
+    context.strategyChange
+  );
```

Comment:
What about when objects are added/deleted/changed?

Reply:
I think the name is misleading... this is a minor util for layout that normally do not change when objects are changed

Reply:
please suggest a better name

Reply:
moved to class